src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -2215,6 +2290,21 @@ void Scene3D::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
   ignmsg << "View angle service on ["
          << this->dataPtr->viewAngleService << "]" << std::endl;
 
+  // view pose service
+  this->dataPtr->moveToPoseService =
+      "/gui/move_to/pose";
+  this->dataPtr->node.Advertise(this->dataPtr->moveToPoseService,
+      &Scene3D::OnMoveToPose, this);
+  ignmsg << "Move to pose service on ["
+         << this->dataPtr->moveToPoseService << "]" << std::endl;
+
+  // camera position topic
+  this->dataPtr->cameraPoseTopic = "/gui/camera/pose";
+  this->dataPtr->cameraPosePub =
+    this->dataPtr->node.Advertise<msgs::Pose>(this->dataPtr->cameraPoseTopic);
+  ignmsg << "Camera pose topic published on ["
```

Comment:
published -> advertised?

Reply:
Fixed in 251b573c3e83ee444d4d136a08eeb5e514eae91f